core/src/main/java/org/bitcoinj/core/DumpedPrivateKey.java

```diff
@@ -86,20 +79,25 @@ public DumpedPrivateKey(@Nullable NetworkParameters params, String encoded) thro
      * Returns an ECKey created from this encoded private key.
      */
     public ECKey getKey() {
-        final ECKey key = ECKey.fromPrivate(bytes);
-        return compressed ? key : key.decompress();
+        final ECKey key = ECKey.fromPrivate(Arrays.copyOf(bytes, 32));
```

Comment:
Rather than creating a compressed key and then decompressing it, shall we use the `ECKey.fromPrivate(bytes, compressed)` constructor in the first place?

Reply:
Sure, of course!